packages/carbon-web-components/src/components/multi-select/multi-select.ts

```diff
@@ -328,6 +374,24 @@ class CDSMultiSelect extends CDSDropdown {
   @property({ attribute: 'clear-selection-label' })
   clearSelectionLabel = '';
 
+  /**
+   * Specify the text that should be read for screen readers that describes total items selected
+   */
+  @property({ attribute: 'clear-selection-description' })
+  clearSelectionDescription = 'Total items selected: ';
+
+  /**
+   * Specify the text that should be read for screen readers to clear selection.
+   */
+  @property({ attribute: 'clear-selection-text' })
+  clearSelectionText = 'To clear selection, press Delete or Backspace.';
+
+  /**
+   * Specify the locale of the control. Used for the default compareItems used for sorting the list of items in the control.
+   */
+  @property()
+  locale = 'en';
```

Comment:
Should this be an attribute so it can be user defined?

Reply:
user should be able to define it, I didn't add that "attribute" field since the attribute name matches